xml/System.IO.Pipelines/PipeScheduler.xml

```diff
@@ -35,7 +35,7 @@
       </AssemblyInfo>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Initializes new a <see cref="T:System.IO.Pipelines.PipeScheduler" /> instance.</summary>
         <remarks>To be added.</remarks>
```

Comment:
If we're not adding remarks now, should we just remove the tags?

Reply:
No, they need to stay, with the `To be added` string inside because it is their default value.